src/login/logind-session-device.c

```diff
@@ -107,17 +107,15 @@ static int session_device_notify(SessionDevice *sd, enum SessionDeviceNotificati
 }
 
 static int sd_eviocrevoke(int fd) {
-        static bool warned;
-        int r;
+        static bool warned = false;
```

Comment:
Patch comment suggests that `warned` wasn't initialized properly, but that's not the case. It was initialized implicitly. So the rework is fine, but the commit message is misleading ;)